services/headless-lms/migrations/20240916122556_course_accesses.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE course_accesses (
+  id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+  course_instance_id UUID NOT NULL REFERENCES course_instances(id),
+  user_id UUID NOT NULL REFERENCES users(id),
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  deleted_at TIMESTAMP WITH TIME ZONE
+);
+COMMENT ON TABLE course_accesses IS 'This table is used to check if user has access to a course that is only joinable by a special code';
```

Comment:
On this comment explain how this table is related to the course_instances.join_code.